build/Targets/Tools.props

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <dotnetRuntimeVersion>2.0.0</dotnetRuntimeVersion>
+    <dotnetRuntimeVersion>2.0.7</dotnetRuntimeVersion>
```

Comment:
:memo: This fixes the CI issue with tests making use of `System.Threading.Tasks.Extensions.dll`, that would try to load version 4.1.1.0 of the assembly rather than the requested 4.2.0 version (likely a runtime assembly  loading bug in `AssemblyLoadContext` which was fixed somewhere between 2.0.0 and 2.0.7).

Thanks @jaredpar @agocke for the help figuring this out! This Friday's donuts/pastries are on me :-)

Andy, if you don't mind, let's keep you machine in current state until we hear back from runtime folks (in case they want to investigate on it).